academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,116 @@
+
+In this tutorial, the sample private library publication records is minimal, with each record only containing a paper title and fake library URL. The goal of the tutorial is to augment the minimal library paper records with additional metadata from MAKES, enabling a robust semantic/keyword search experience.
+
+In the sample entity linking Powershell script, we will leverage paper entity's DOI, citation count, abstract, fields of study, authors, affiliations, journal, and conference information to enrich our data and support semantic/keyword search.
+
+We retrieve these information by constructing an Interpret request foreach link attempt with those attributes as the following:
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_interpret_request":::
+
+Once we're able to link a library paper against a MAKES paper entity successfully, we will then merge the entities together using the following helper function:
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_merge_entities":::
+
+## Run sample solution to see entity linking in action
+
+1. Deploy a MAKES instance by following [Create API Instance](get-started-create-api-instances.md) guide
+1. Download [Sample Library Data](samplePrivateLibraryData.json)
+1. Download [Sample Entity Linking Script](linkPrivateLibraryData.ps1)
+1. Modify the entity linking script and set the **makesEndpoint** variable to your deployed MAKES instance url
+1. Open up a powershell console and run the sample entity linking script. Make sure the sample library data and linking script are under the same directory.
+
+The sample entity linking script output should look like the following:
+
+Linked | ConfidenceScore | OriginalTitle | MagTitle
+---- | ---- |---- |----
+True | 0 | Microsoft Academic Graph: When experts are not enough | Microsoft Academic Graph: When experts are not enough                                                                                                                     True | 0 | An Overview of Microsoft Academic Service (MAS) and Applications | An Overview of Microsoft Academic Service (MAS) and Applications                                                                                               False | -137 | The impact of support system of enterprise organization on Taiwanese expatriates’ work-family conflict | Koha enterprise resource planning system and its potential impact on information management organizations            True | 0 | Organisational justice and customer citizenship behaviour of retail industries | Organisational justice and customer citizenship behaviour of retail industries                                                                   True | 0 | A Scalable Hybrid Research Paper Recommender System for Microsoft Academic | A Scalable Hybrid Research Paper Recommender System for Microsoft Academic
```

Comment:
>                                                                                                                     True | 0 | An Overview of Microsoft Academic Service (MAS) and Applications | An Overview of Microsoft Academic Service (MAS) and Applications                                                                                               False | -137 | The impact of support system of enterprise organization on Taiwanese expatriates’ work-family conflict | Koha enterprise resource planning system and its potential impact on information management organizations            True | 0 | Organisational justice and customer citizenship behaviour of retail industries | Organisational justice and customer citizenship behaviour of retail industries                                                                   True | 0 | A Scalable Hybrid Research Paper Recommender System for Microsoft Academic | A Scalable Hybrid Research Paper Recommender System for Microsoft Academic [](start = 120, length = 990)

Looks like a number of records ended up on the same line, please fix

Reply:
Completely broken now. Please make sure you double check the test site ([https://review.docs.microsoft.com/en-us/academic-services/knowledge-exploration-service/tutorial-entity-linking?branch=devs/alch/entity-linking-tutorial-v2#run-sample-solution-to-see-entity-linking-in-action](https://review.docs.microsoft.com/en-us/academic-services/knowledge-exploration-service/tutorial-entity-linking?branch=devs/alch/entity-linking-tutorial-v2#run-sample-solution-to-see-entity-linking-in-action)) after checking in changes.

---
In reply to: [508735270](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/185#discussion_r508735270) [](ancestors = 508735270)